### DISCUSSION TITLE
Bulk import or merge CAMT 053 files

### DISCUSSION THREAD

**Discussion by beune** [firefly-iii/firefly-iii#8105] [Firefly III] [answered]
My bank (ABN AMRO) allows for exporting CAMT053 files. However, the output is a zip file containing hundreds of CAMT 053 files. Is there a way to merge those files into one, or to upload them in bulk using the firefly data importer?

I do not want to have to import +300 manually, I hope you understand 😃

**Comment by JC5** [comment]
There is a todo on the list to allow the import of zip files, but until then you can try running the same command on all files.

Something like this?

https://unix.stackexchange.com/questions/80851/how-do-i-run-a-command-on-multiple-files

**Comment by JC5** [comment, marked as answer]
The command can be found here:

https://docs.firefly-iii.org/data-importer/advanced/automation/#automated-imports-over-the-command-line
